cibuildwheel/windows.py

```diff
@@ -22,6 +28,12 @@ def get_python_path(config):
             return glob(path_pattern)[0]
         except IndexError:
             raise Exception('Could not find a Python install at ' + path_pattern)
+    elif IS_RUNNING_ON_TRAVIS:
+        if config.version == "3.4.x":
+            return config.path
```

Comment:
Better to return `None` here I think

Reply:
This case allow someone, who really  python 3.4 to install it manually and use. The result of this function in rest of code is not checked and is treated as string. I think that it is better to left this or literally raise the exception than return `None`

Reply:
Okay, fair enough. Btw, do you have a specific use case for Python 3.4? We're going to be removing it very soon from cibuildwheel since it hit end-of-life this year (#168)

Reply:
No. I only use python 3.6+.